### PR TITLE
update ontology

### PR DESCRIPTION
**Pull request by Gouga34**
Add radiometric target concepts and relations : 
- properties subclassof deviceProperty
- document type : SpectralHemisphericDirectionalReflectanceDocument
- and minor infrastructure labels corrections

### CODE REVIEW

File: oepo.owl [hunk 1]
```diff
@@ -1631,8 +1888,28 @@ Peut définir une zone au sens physique ou théorique</rdfs:comment>
...
+1 ligne d’en-tête : première colonne = « id » et colonnes suivantes les longueurs d’onde en nm que l’on peut rencontrer durant les traitements
+caractère de séparation : tabulation
+id : alias
+pourcentage de réflectance mesuré pour chaque longueur d’onde</rdfs:comment>
+        <rdfs:comment xml:lang="en">csv file with the following format : 
+1 header line : first column = &quot;id&quot;. The following columns are the wavelength (nm) that could be founded during the treatments
+separator : tabular
+id : alias
+measured spectral hemispheric directional reflectance percent for each wavelength</rdfs:comment>
+        <rdfs:label xml:lang="fr">Réflectance directionnelle hémisphérique spectrale</rdfs:label>
```
Comment (annetireau) [inline oepo.owl]: Please add "Fichier de réflect..."

File: oepo.owl [hunk 2]
```diff
@@ -1631,8 +1888,28 @@ Peut définir une zone au sens physique ou théorique</rdfs:comment>
...
+        <rdfs:label xml:lang="en">Spectral Hemispheric Directional Reflectance</rdfs:label>
```
Comment (annetireau) [inline oepo.owl]: Please add "File" in the end

File: oepo.owl [hunk 3]
```diff
@@ -405,29 +405,73 @@ The Object Ontology is licensed by CC 4.0 (http://creativecommons.org/licenses/b
...
+    
 
 
+    <!-- http://www.phenome-fppn.fr/vocabulary/2017#brdfP4 -->
+
+    <owl:DatatypeProperty rdf:about="http://www.phenome-fppn.fr/vocabulary/2017#brdfP4">
+        <rdfs:subPropertyOf rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#deviceProperty"/>
+        <rdfs:domain rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#RadiometricTarget"/>
+        <rdfs:comment xml:lang="en">BRDF(θ)=P1+P2cos(θ)+1/2*P3(3*cos^2θ−1)+1/2*P4(5*cos^3θ−3*cosθ)</rdfs:comment>
+        <rdfs:label xml:lang="en">BRDF P4</rdfs:label>
```
Comment (annetireau) [inline oepo.owl:454]: Please remove language attribut  (xml:lang="en") for all BRDF

File: oepo.owl [hunk 4]
```diff
@@ -405,29 +405,73 @@ The Object Ontology is licensed by CC 4.0 (http://creativecommons.org/licenses/b
...
+    </owl:DatatypeProperty>
+    
 
 
+    <!-- http://www.phenome-fppn.fr/vocabulary/2017#brdfP4 -->
+
+    <owl:DatatypeProperty rdf:about="http://www.phenome-fppn.fr/vocabulary/2017#brdfP4">
+        <rdfs:subPropertyOf rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#deviceProperty"/>
+        <rdfs:domain rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#RadiometricTarget"/>
+        <rdfs:comment xml:lang="en">BRDF(θ)=P1+P2cos(θ)+1/2*P3(3*cos^2θ−1)+1/2*P4(5*cos^3θ−3*cosθ)</rdfs:comment>
```
Comment (annetireau) [inline oepo.owl:452]: Great !!

File: oepo.owl [hunk 5]
```diff
@@ -484,35 +528,132 @@ The Object Ontology is licensed by CC 4.0 (http://creativecommons.org/licenses/b
...
+    
 
 
     <!-- http://www.phenome-fppn.fr/vocabulary/2017#hasExperimentModalities -->
 
     <owl:DatatypeProperty rdf:about="http://www.phenome-fppn.fr/vocabulary/2017#hasExperimentModalities"/>
+    
+
+
+    <!-- http://www.phenome-fppn.fr/vocabulary/2017#hasFigure -->
```
Comment (annetireau) [inline oepo.owl]: I think you should use "shape" instead or "figure"

File: oepo.owl [hunk 6]
```diff
@@ -484,35 +528,132 @@ The Object Ontology is licensed by CC 4.0 (http://creativecommons.org/licenses/b
...
+                        <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#List"/>
+                        <rdf:first>Carpet</rdf:first>
+                        <rdf:rest>
+                            <rdf:Description>
+                                <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#List"/>
+                                <rdf:first>Paint</rdf:first>
+                                <rdf:rest>
+                                    <rdf:Description>
+                                        <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#List"/>
+                                        <rdf:first>Spectralon</rdf:first>
```
Comment (annetireau) [inline oepo.owl]: Is "spectralon" a specific word ?

File: oepo.owl [hunk 7]
```diff
@@ -484,35 +528,132 @@ The Object Ontology is licensed by CC 4.0 (http://creativecommons.org/licenses/b
...
+        <rdfs:domain rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#RadiometricTarget"/>
+        <rdfs:label xml:lang="fr">Largeur du rectangle</rdfs:label>
+        <rdfs:label xml:lang="en">Rectangle width</rdfs:label>
+    </owl:DatatypeProperty>
+    
+
+
+    <!-- http://www.phenome-fppn.fr/vocabulary/2017#hasMaterial -->
+
+    <owl:DatatypeProperty rdf:about="http://www.phenome-fppn.fr/vocabulary/2017#hasMaterial">
```
Comment (annetireau) [inline oepo.owl]: Do you really whant to use "has" prefix for the name of this properties ?

Comment (annetireau) [inline oepo.owl]: hasMaterial is to generic for me, may be add hasRadiometricTargetMaterial.

File: oepo.owl [hunk 8]
```diff
@@ -1972,157 +2249,34 @@ Peut définir une zone au sens physique ou théorique</rdfs:comment>
...
     <!-- http://www.phenome-fppn.fr/vocabulary/2017#c -->
 
     <owl:NamedIndividual rdf:about="http://www.phenome-fppn.fr/vocabulary/2017#c">
         <rdf:type rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#Camera"/>
     </owl:NamedIndividual>
-
+    
 
 
     <!-- http://www.phenome-fppn.fr/vocabulary/2017#tc -->
```
Comment (annetireau) [inline oepo.owl]: I think you should remove this instance

File: oepo.owl [hunk 9]
```diff
@@ -1972,157 +2249,34 @@ Peut définir une zone au sens physique ou théorique</rdfs:comment>
...
     // Individuals
     //
     ///////////////////////////////////////////////////////////////////////////////////////
      -->
 
-
+    
 
 
     <!-- http://www.phenome-fppn.fr/vocabulary/2017#c -->
```
Comment (annetireau) [inline oepo.owl]: you should remove this instance

File: oepo.owl [hunk 10]
```diff
@@ -1631,8 +1888,28 @@ Peut définir une zone au sens physique ou théorique</rdfs:comment>
...
+
+    <owl:Class rdf:about="http://www.phenome-fppn.fr/vocabulary/2017#SpectralHemisphericDirectionalReflectanceDocument">
+        <rdfs:subClassOf rdf:resource="http://www.phenome-fppn.fr/vocabulary/2017#Document"/>
+        <rdfs:comment xml:lang="fr">Fichier texte au format csv suivant :
+1 ligne d’en-tête : première colonne = « id » et colonnes suivantes les longueurs d’onde en nm que l’on peut rencontrer durant les traitements
+caractère de séparation : tabulation
+id : alias
+pourcentage de réflectance mesuré pour chaque longueur d’onde</rdfs:comment>
+        <rdfs:comment xml:lang="en">csv file with the following format : 
+1 header line : first column = &quot;id&quot;. The following columns are the wavelength (nm) that could be founded during the treatments
```
Comment (annetireau) [inline oepo.owl:1910]: Nice !!

**Review by annetireau — CHANGES_REQUESTED** [review]
I think we need to ling installation and radiometrictarget -> to discuss

File: oepo.owl (hunk 3 above)
Comment (annetireau) [reply oepo.owl:454]: Add comment

Comment (annetireau) [reply oepo.owl:454]: Bidirectional reflectance distribution function

**Comment by Gouga34** [comment]
Requested updates done

**Pull request merged**